### ISSUE TITLE
Cannot find disk after reboot step for RT Kernel

### ISSUE THREAD

**Issue by adimehrotra** [raess1/UPxtreme-RT-kernel#1]
Hello! I've been trying this on my UpBoard Xtreme trying to get the RT Kernel running, I followed the steps in the README but when I got to "sudo reboot" after the "grub update" the system did reboot into the new kernel but entered "initramfs" saying it cannot find the root partition? Any ideas as to a fix?

I have checked the UUID the kernel is searching for with the UUID of the actual disk from the original 5.4 kernel and they match, I think it may be missing a driver but I'm not sure.

**Comment by raess1** [comment]
Is this ubuntu 18.04? did you set the CONFIG_AUFS_FS=n ?


best regards
Robin

**Comment by adimehrotra** [comment]
Thanks for the response! It is ubuntu 18.04. I have not tired CONFIG_AUFS_FS=n. In the instructions it says "turn off if make fails?" - may I ask what exactly does turning this on and off do?

**Comment by raess1** [comment]
Another user that did successfully build the kernel without `CONFIG_AUFS_FS=n` also experience simlar issues as you. 
So please try to set it as I suggested and try.

In my case,  I can not even build the kernel withot this option. :)

Let me know

**Comment by adimehrotra** [comment]
I have now set the  CONFIG_AUFS_FS=n but this is what I'm getting on the output? It's still going to initramfs? Anything else I could be missing?

![Image from iOS](https://user-images.githubusercontent.com/46380543/117321214-f07dd480-ae5a-11eb-8d28-b165e559dd27.jpg)

**Comment by adimehrotra** [comment]
Here is the config file if this helps? Maybe I'm just doing something silly? 
[CONFIG_RT.txt](https://github.com/raess1/UPxtreme-RT-kernel/files/6435735/CONFIG_RT.txt)

**Comment by raess1** [comment]
I will check in a while and compare it to my cfg :)

Den tors 6 maj 2021 17:22Aditya Mehrotra ***@***.***> skrev:

**Comment by adimehrotra** [comment]
Thank you so much for helping with this in advance :) !

**Comment by dmarew** [comment]
I am facing the same issue. Have you been able to fix it?

**Comment by elpimous** [comment]
@dmarew, did you first installed specific up xtreme kernel, as mentionnés on up Ubuntu wiki?
And you do the RT process after..
I had same error, flashing standard kernel...

**Comment by dmarew** [comment]
@elpimous Yes, I installed the standard upboard-ubuntu with the linux kernel 5.4.0-1-generic.  Were you able to fix the issue?

**Comment by elpimous** [comment]
@dmarew, another problem I had was an error in compilation. Even if final compilation success. 
Did you read carefully All build/make lines?

**Comment by dmarew** [comment]
I didn't get any compilation errors just a few warnings and no I haven't gone through the makefile. I probably should.

**Comment by elpimous** [comment]
If no error, on each command line, and up kernel, I don t see, sorry.
I'd restart all the patch, looking at each code feedback, and perhaps use the sudo command?!
Good luck

**Comment by dmarew** [comment]
Thanks. But where able to successfully install it? Can you send me the .config file you used?
